src/pages/home/home.stories.js

```diff
@@ -403,46 +403,46 @@ const Template = (args) => (
             title-text="Help &amp; Contact."
             variant="positive"
           >
-            <li class="link-list__item" slot="link-list__item">
-              <sbb-link
-                href-value="https://www.sbb.ch/de/hilfe-und-kontakt/erstattung-entschaedigung/rueckerstattung-von-billetten.html"
-                text="Rückerstattungen"
-                text-size="s"
-                variant="positive"
-              ></sbb-link>
-            </li>
-            <li class="link-list__item" slot="link-list__item">
-              <sbb-link
-                href-value="https://www.sbb.ch/de/hilfe-und-kontakt/erstattung-entschaedigung/rueckerstattung-von-billetten.html"
-                text="Fundbüro"
-                text-size="s"
-                variant="positive"
-              ></sbb-link>
-            </li>
-            <li class="link-list__item" slot="link-list__item">
-              <sbb-link
-                href-value="https://www.sbb.ch/de/hilfe-und-kontakt/erstattung-entschaedigung/rueckerstattung-von-billetten.html"
-                text="Beschwerden"
-                text-size="s"
-                variant="positive"
-              ></sbb-link>
-            </li>
-            <li class="link-list__item" slot="link-list__item">
-              <sbb-link
-                href-value="https://www.sbb.ch/de/hilfe-und-kontakt/erstattung-entschaedigung/rueckerstattung-von-billetten.html"
-                text="Lob aussprechen"
-                text-size="s"
-                variant="positive"
-              ></sbb-link>
-            </li>
-            <li class="link-list__item" slot="link-list__item">
-              <sbb-link
-                href-value="https://www.sbb.ch/de/hilfe-und-kontakt/erstattung-entschaedigung/rueckerstattung-von-billetten.html"
-                text="Sachbeschädigung melden"
-                text-size="s"
-                variant="positive"
-              ></sbb-link>
-            </li>
+            <sbb-link
```

Comment:
All links in the story have an old api. text is now the default slot, variant="positive" is not existing (default), href-value should be "href" only.